brainstormforce/astra
# AST-3611 - improvement: Introducing new vw unit support for font-size in customizer typography settings

### Description
- Introducing new vw unit support for font-size in customizer typography settings
- Limit set to 25vw

### Screenshots
- https://d.pr/i/Erl7Ez 

### Types of changes
- New feature (non-breaking change)

### How has this been tested?
- Get an overview of all section's typography settings
- Check some settings on the frontend
- Check the working preview in customizer

### Checklist:
